src/bridge/bridge.c

```diff
@@ -62,7 +62,7 @@
    of the user that is logged into the Server Console.
 */
 
-static CockpitPackages *packages;
+static CockpitPackages *packages = NULL;
```

Comment:
For the record: This is unnecessary (but harmless), static variables are always initialized to 0.